config/default/kustomization.yaml

```diff
@@ -71,3 +69,7 @@ vars:
 #    kind: Service
 #    version: v1
 #    name: webhook-service
+resources:
```

Comment:
lol, I feel like this has gone full circle...  we started with resources, then moved to bases, and now back to resources again :joy: